samples/setting-up-dependency-injection/Startup.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Net.Http;
+using EventStore.Client;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace setting_up_dependency_injection {
+	public class Startup {
+		public Startup(IConfiguration configuration) {
+			Configuration = configuration;
+		}
+
+		public IConfiguration Configuration { get; }
+
+		public void ConfigureServices(IServiceCollection services) {
+			services.AddControllers();
+
+			#region setting-up-dependency
+			services.AddEventStoreClient(settings => {
```

Comment:
The latest bits have an overload for connection string, I think we should show that here as well.

Reply:
Updated to connection string usage in all samples.